spec/services/hyrax/admin_set_create_service_spec.rb

```diff
@@ -292,8 +292,7 @@
         let(:admin_set) { FactoryBot.build(:invalid_hyrax_admin_set) } # Missing title
 
         it 'will not call the workflow_importer' do
-          expect { service.create! }.to raise_error(RuntimeError)
-          expect(workflow_importer).not_to have_received(:call)
+          expect { service.create! }.to raise_error(ActiveRecord::RecordNotFound)
```

Comment:
I removed it because at the time I was testing it...it was no longer valid. related to the note below.

if I recall correctly...since we weren't validating in `Hyrax::Persister#save`, any longer, we did not get the runtime error anymore from `Hyrax::AdminSetCreateService#create!`. therefore the workflow importer _was_ called. later down the stacktrace in `Sipity::Workflow#activate!`, `Sipity::Workflow.find_by!` failed because we didn't have a `workflow_id`. that caused the `ActiveRecord::RecordNotFound` we're now checking for.


when I spoke to Rob about it, the different error and change in the `save` method, he said to change the error we were now expecting. I did not however explain the full background of the spec with him.

now that I'm passing a `perform_af_validation` param to `save` though...I can pass `perform_af_validation: true` inside `Hyrax::AdminSetCreateService#create!`.
hopefully that doesn't break another spec. 😩 🤞🏾 

<details>
<summary>note</summary>

Tuesday, August 1, 2023
while pairing last week, rob suggested [this change](https://github.com/samvera/hyrax/pull/6127/commits/9b64c54761216dcef982e2157539638662faa90b). I noticed last night that committing that change made the [7 failing works_controller_behavior_spec’s](https://app.circleci.com/pipelines/github/samvera/hyrax/9332/workflows/eb885c03-e719-41ba-8d71-855a3e8c35de/jobs/72774/tests), pass.
a little bit ago however, I noticed that the change caused spec/services/hyrax/admin_set_create_service_spec.rb:294 to fail.
I’m now unsure whether I should or should not use validate: false in [Wings::Valkyrie::Persister#save](https://github.com/samvera/hyrax/blob/9b64c54761216dcef982e2157539638662faa90b/lib/wings/valkyrie/persister.rb#L26). or the best practice way forward on making sure that the specs in both files pass…and that we’re doing the right thing.
</details>

Reply:
my proposed change broke several things because `Hyrax.persister.save` that gets called inside of `Hyrax::AdminSetCreateService#create!` is not the same as `Wings::Persister#save`. I've reverted my change and updated the assertion.

ref: https://github.com/samvera/hyrax/pull/6127/commits/f992e109d1903a48407a15840f45f9bd2ecef3f2